packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -150,6 +150,51 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ** See code in examples/api/lib/widgets/gesture_detector/gesture_detector.1.dart **
 /// {@end-tool}
 ///
+/// ## Troubleshooting
```

Comment:
This section should probably somewhere link to https://docs.flutter.dev/development/ui/advanced/gestures#gesture-disambiguation as background reading.